Python/Currency-Converter/converter.py

```diff
@@ -0,0 +1,18 @@
+import requests
+api = "https://api.exchangerate-api.com/v4/latest/USD"
+# Getting Currency Rates from the API
+rates = requests.get(api).json()['rates']
+# Currency from which we need to convert
+from_currency = input("Enter the currency code from which you want to convert : ").upper()
```

Comment:
you need to give suggestions for currency code to the user to enter. How to write a currency code?